src/app/auth/sign-in/sign-in.page.ts

```
@@ -160,18 +160,37 @@ export class SignInPage implements OnInit {
-      header = 'Temporary Lockout';
+    switch (error?.status) {
+      case 400:
+        this.trackingService.eventTrack('Go to Invite Expired page');
+        this.router.navigate(['/', 'auth', 'pending_verification', { email: this.fg.controls.email.value as string }]);
+        break;
+
+      case 406:
+        this.trackingService.eventTrack('Go to Password Expired page');
+        const queryParams: Record<string, boolean> = {
+          tmp_pwd_expired: true,
+        };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)